usr.sbin/adduser/adduser.sh

```diff
@@ -613,6 +654,84 @@ get_password() {
 	fi
 }
 
+# get_zfs_encryption
+#	Read ZFS options for the user's home dataset.
+#
+get_zfs_encryption() {
+	_input=
+	_prompt="Enable ZFS encryption? (yes/no) [no]: "
+	while : ; do
+		echo -n "$_prompt"
+		read _input
+
+		[ -z "$_input" ] && _input=Zencrypt
+		case $_input in
+		[Nn][Oo]|[Nn])
+			Zencrypt=
+			break
+			;;
+		[Yy][Ee][Ss]|[Yy][Ee]|[Yy])
```

Comment:
I think accepting "ye" is a little nonstandard; this should probably just be a case-insensitive "yes" or case-insensitive "y"

Reply:
I agree, it's a bit strange.  However, I followed the user input convention already used in the script.

Reply:
Huh, indeed- funky